seed/lib/superperms/orgs/models.py

```diff
@@ -39,6 +39,18 @@
     (STATUS_REJECTED, 'Rejected'),
 )
 
+MEASUREMENT_CHOICES_AREA = (
+    ('ft**2', 'square feet'),
+    ('m**2', 'square metres'),
+)
+
+MEASUREMENT_CHOICES_EUI = (
```

Comment:
can you scope this to be part of the OrganizationUser so it isn't just hanging out there on its own? Something like:

```
class OrganizationUser:
    MEASUREMENT_CHOICES_EUI = (...)
```